packages/app-content-pages/src/screens/Publications/Publications.js

```diff
@@ -55,21 +50,22 @@ function Publications ({
   )
 
   const sidebar = (
-    <Box as='ul' gap='small'>
+    <Box as='menu' gap='small' name='Filter by category'>
```

Comment:
Menu is deprecated in HTML. I’d leave this as `<ul>` for screenreaders.

Reply:
Sorry `<menuitem>` is deprecated in HTML, but I think `<ul>` might have better support than `<menu>` for marking up a list of links. See also #667.

Reply:
Okay will revert